daily-challenges/2024-09/17/haquyen.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { test, expect } from "@playwright/test";
+
+function isPrime(num: number): boolean {
+  if (num <= 1) return false;
```

Comment:
Nên có ngoặc nhọn trong block code if, else, for,... dù chỉ có 1 dòng code.
Bạn có thể đọc thêm 1 bài viết vui về phần này: https://minhphong306.wordpress.com/2023/08/24/tai-sao-luon-nen-them-ngoac-nhon-du-chi-co-1-dong-code-sau-if-for/

Reply:
hi anh, em nghi trong case nay khong nhat thiet phai use block form. this condition and action are kinda simple and it will not grow. so single-line is the best in this scenario

Reply:
Yep. Just my suggestion ^^